README.md

```diff
@@ -905,8 +905,12 @@ to_json("<sourceField>"[, pretty: "<boolean>"][, error_string: "<errorValue>"])
 
 Replaces the value with its Base64 encoding.
 
+Options:
+
+-`url_safe`: Whether to encode a URL. (Default: `false`)
```

Comment:
Maybe "borrow" the description from [base64encode.org](https://www.base64encode.org/)?

```suggestion
-`url_safe`: Perform URL-safe encoding (uses Base64URL format). (Default: `false`)
```

Reply:
yes, better :)